platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -480,25 +329,10 @@ public class SwirldsPlatform implements Platform, ConnectionTracker, Startable {
         registerAddressBookMetrics(metrics, initialAddressBook, selfId);
 
         this.consensusMetrics = new ConsensusMetricsImpl(this.selfId, metrics);
-        this.eventIntakeMetrics = new EventIntakeMetrics(metrics, time);
-        this.syncMetrics = new SyncMetrics(metrics);
-        this.networkMetrics = new NetworkMetrics(metrics, selfId, getAddressBook());
-        metrics.addUpdater(networkMetrics::update);
-        this.reconnectMetrics = new ReconnectMetrics(metrics);
-        RuntimeMetrics.setup(metrics);
 
-        final ChatterConfig chatterConfig = platformContext.getConfiguration().getConfigData(ChatterConfig.class);
-        if (chatterConfig.useChatter()) {
-            chatterCore = new ChatterCore<>(
-                    time,
-                    GossipEvent.class,
-                    new PrepareChatterEvent(CryptographyHolder.get()),
-                    chatterConfig,
-                    networkMetrics::recordPingTime,
-                    metrics);
-        } else {
-            chatterCore = null;
-        }
+        EventIntakeMetrics eventIntakeMetrics = new EventIntakeMetrics(metrics, time);
```

Comment:
can these new variables be made final?

Reply:
Yes, fixed. I blame intellij. It tells you when a classmember variable can be made into a local variable, but if you use the automatic refactoring it drops `final`.